src/guides/v2.3/config-guide/deployment/pipeline/technical-details.md

```diff
@@ -112,6 +112,19 @@ On your development system:
 
 1. Add your modified code and the shared configuration to source control.
 
+1. For clear generated php code, static assets files while in development developers can use commands below.
+
+   ```bash
+      rm -rf var/view_preprocessed/*
+      rm -rf pub/static/frontend/* pub/static/adminhtml/*
```

Comment:
```suggestion
      rm -rf pub/static/*/* 
```

Reply:
And also remove the `-f` flag. It's bad practice to teach people to use this flag, it should only be used in very exceptional cases.